materialize-sqlserver/.snapshots/TestSQLGeneration

```diff
@@ -34,17 +34,13 @@ TRUNCATE TABLE #flow_temp_store_1;
 
 CREATE TABLE #flow_temp_load_0 (
 		key1 BIGINT NOT NULL,
-		"key!2" BIT NOT NULL,
-		PRIMARY KEY (key1, "key!2")
```

Comment:
Was there something not working with this constraint in place? We should only see a key a single time per load transaction. If there were conflicts, it may be indicative of something else going on.

Reply:
@williamhbaker yes, we were getting some errors about this. Then this is fishy.  I'll investigate this 👀

Reply:
@williamhbaker this was due to a case insensitive collation on the table, I think we need to explicitly set the collation in SQLServer to be case-sensitive as well (and utf8). I'll do that.